main.go

```diff
@@ -193,8 +199,35 @@ func main() {
 		logger.Error("unable to create controller", zap.String("controller", "NifiNodeGroupAutoscaler"), zap.Error(err))
 		os.Exit(1)
 	}
-	// +kubebuilder:scaffold:builder
 
+	if webhookEnabled {
```

Comment:
I think you've missed the `NifiNodeGroupAutoscaler` here.

Reply:
Not necessary because we only manage the v1alpha1 version for this resource!